app/controllers/api/functions.php

```diff
@@ -474,6 +475,38 @@
         ])));
 
         if ($next) {
```

Comment:
We might want to deprecate this attribute as it can be an overkill to update the function document on every execution. We can just calculate the date in runtime for the HTTP response.

Reply:
Instead we can just check $schedule here.